src/item.cpp

```diff
@@ -6953,6 +7078,21 @@ bool item::process_food( const player *carrier, const tripoint &p, int temp, flo
     return false;
 }
 
+bool item::process_craft( const player *carrier, const tripoint &p, int temp, float insulation )
+{
+    if( carrier != nullptr && carrier->has_item( *this ) ) {
+        temp += 5; // body heat increases inventory temperature
+        insulation *= 1.5; // clothing provides inventory some level of insulation
+    }
+
+    // Process all stored components
+    // Already applied body heat, so pass nullptr instead of carrier
```

Comment:
Why? The upper snippet is copy-pasted verbatim.

Reply:
No idea, guess I was sleepy.